airbyte-integrations/connectors/destination-e2e-test/build.gradle

```diff
@@ -1,4 +1,5 @@
 plugins {
+    id 'airbyte-java-connector'
```

Comment:
this is a hack. We get the `integrationTestJava` task from this plugin. https://github.com/airbytehq/airbyte-internal-issues/issues/9864 will fix.